src/stores/project.store.ts

```diff
@@ -48,10 +52,31 @@ export class ProjectStore {
     return `${instrumentName} ${instrumentCount + 1}`;
   }
 
-  private getSheet(sheetId: SheetId) {
+  public getSheet(sheetId: SheetId) {
     return this.sheetList.find(sheet => sheet.id === sheetId);
   }
 
+  public getAdjacentSheets(
+    sheetId: SheetId
+  ): { prevSheet: SheetId | null; nextSheet: SheetId | null } {
```

Comment:
q for you: I usually haven't been explicitly spelling out return types like this. Do you think it's helpful/needed or otherwise valuable for documentation etc.?

Reply:
So it's definitely necessary for TS to work since the TypeScript compiler can infer the implicit return types. But it maybe useful for documentation? Although its just more code to update if you do end up changing your function 😛